localization-proofreading-english/StringTableEnExaminations.xml

```diff
@@ -101,76 +101,76 @@
+            <GameDBLocalizedString>      <LocID>EXM_SEROLOGY_TESTING_SAMPLING</LocID>                           <Text>Serologic test - sampling</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>EXM_SEROLOGY_TESTING_SAMPLING_DESCRIPTION</LocID>               <Text>Drawing of a blood sample for serologic tests. \n\nRequired room: Stat Lab \n\nRequired equipment: Ultracentrifuge + RIA</Text>  </GameDBLocalizedString>
             <GameDBLocalizedString>      <LocID>EXM_SEROLOGY_TESTING_TESTING</LocID>                            <Text>Serologic test - testing</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>EXM_SEROLOGY_TESTING_TESTING_DESCRIPTION</LocID>                <Text>It is a special blood test, looking for specific antibodies and antigens, which are unique proteins made by human immune system as reaction to contact with specific viruses, bacterias, diseases. \n\nRequired room: Stat Lab \n\nRequired equipment: Ultracentrifuge + RIA</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>EXM_SKIN_INJECTION_TEST</LocID>                                 <Text>Skin injection test</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>EXM_SKIN_INJECTION_TEST_DESCRIPTION</LocID>                     <Text>Injection of a different small amounts of allergen extracts into the skin, usually on arm to confirm or deny the allergic reaction of the body. \n\nRequired room: Offices / Diagnostic units at Emergency dpt / ICU dpt / Neurology dpt</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>EXM_SEROLOGY_TESTING_TESTING_DESCRIPTION</LocID>                <Text>Special blood test examining specific antibodies and antigens. Antibodies are proteins made by human immune system to neutralize pathogens. \n\nRequired room: Stat Lab \n\nRequired equipment: Ultracentrifuge + RIA</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>EXM_SKIN_INJECTION_TEST</LocID>                                 <Text>Skin allergy test</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>EXM_SKIN_INJECTION_TEST_DESCRIPTION</LocID>                     <Text>Injection of a small amount of different of allergen extracts into the skin usually on the arm to confirm or deny the allergic reactions to specific allergens. \n\nRequired room: Offices / Diagnostic units at Emergency dpt / ICU dpt / Neurology dpt</Text>  </GameDBLocalizedString>
             <GameDBLocalizedString>      <LocID>EXM_SPEECH_LISTENING</LocID>                                    <Text>Speech listening</Text>  </GameDBLocalizedString>
             <GameDBLocalizedString>      <LocID>EXM_SPEECH_LISTENING_DESCRIPTION</LocID>                        <Text>Listening to speech flow of the patient. \n\nRequired room: Any office / Diagnostic unit</Text>  </GameDBLocalizedString>
```

Comment:
Linje 123-124 ved jeg ikke rigtig hvad der menes med - Speech listening. Er det i psykiatrisk sammenhæng, eller hvordan? Har svært ved at differentiere den fra Interview (linje 115).

Reply:
Det er bare en undersøgelse, hvor lægen "lytter til tale". For at fange afasi etc.
'Interview' er nok mere anamnese. I den virkelige verden fanger man jo den slags i anamnesen, men her skal spillet vide hvad det skal 'kigge' efter, derfor skal 'speech listening' aktiveres, hvis nogle af de symptomer skal kunne fanges.